website/docs/d/awp_aws_onboarding_data.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "dome9"
+page_title: "Check Point CloudGuard Dome9: dome9_awp_aws_get_onboarding_data"
```

Comment:
remove "Get". Please, check all places.

Reply:
no currently only for few including agentless_account_settings (since it has an update api)

Reply:
so, what would happen if the user changes one of those other parameters and runs 'terraform apply'?

Reply:
nothing will be changed, once some other params changes that we don't support, terraform apply will not touch the resource or will not call any further apis (since we don't have any apis to change them), so it will not have any effect